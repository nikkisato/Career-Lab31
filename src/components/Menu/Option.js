import React from 'react';
import PropTypes from 'prop-types';

const Option = ({ value, name }) => (
  <>
    <option value={value}>{name}</option>
  </>
);

Option.propTypes = {
  value: PropTypes.string.isRequired,
  name: PropTypes.string.isRequired
};

export default Option;
