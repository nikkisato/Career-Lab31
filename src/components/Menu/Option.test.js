import React from 'react';
import { shallow } from 'enzyme';
import Option from './Option';

describe('Option component', () => {
  it('renders Option', () => {
    const wrapper = shallow(<Option />);
    expect(wrapper).toMatchSnapshot();
  });
});
