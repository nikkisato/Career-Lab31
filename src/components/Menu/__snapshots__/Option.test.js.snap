// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Option component renders Option 1`] = `
<Fragment>
  <option />
</Fragment>
`;
